admin/display/menu.php

```diff
@@ -44,6 +48,27 @@ function add_pro_submenu() {
 			$url,
 		);
 	}
+
+	/*
+	 * Opens Upgrade link in menu as new tab.
+	 *
+	 * @since 3.0
+	 */
+	function upgrade_link_aioseop_menu_new_tab() {
+		wp_enqueue_script( 'aioseop_menu_js', AIOSEOP_PLUGIN_URL . 'js/aioseop-menu.js', array( 'jquery' ), AIOSEOP_VERSION, true );
```

Comment:
Can these be enqueued in the footer in conjunction to the anonymous function (false on 5th param).

Reply:
Done.

Reply:
Disregard, true is for footer